includes/sanitizers/class-amp-img-sanitizer.php

```diff
@@ -352,6 +346,13 @@ private function adjust_and_replace_node( DOMElement $node ) {
 			foreach ( $attributes as $name => $value ) {
 				$node->setAttribute( $name, $value );
 			}
+
+			// Mark element as PX-verified to prevent raising validation errors for an intentionally invalid <img>.
+			// It doesn't make sense to raise a validation error to allow the user to decide whether to convert from
+			// <img> to <amp-img> since the native_img_used arg is the opt-in to not do any such conversion.
+			// @todo Remove once https://github.com/ampproject/amphtml/issues/30442 lands.
+			ValidationExemption::mark_node_as_px_verified( $node );
```

Comment:
@pierlon Same here:

![image](https://user-images.githubusercontent.com/134745/135370568-53e13579-93fa-44cb-ada3-232e11a6450f.png)

This code _is_ getting run as part of `AMP_Img_Sanitizer_Test::test_converter` for the datasets `simple_native_image` and `native_image_with_no_dims_and_loading`.

Reply:
Hmm yea I see what you mean... looking into this.